plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -391,7 +306,7 @@ export class SonataFlowService {
   private extractWorkflowType(
     workflowDef: WorkflowDefinition,
   ): string | undefined {
-    if (workflowDef.annotations) {
+    if (workflowDef && workflowDef.annotations) {
```

Comment:
I think this change will raise a code smell issue. Please consider the following:

```suggestion
    if (workflowDef?.annotations) {
```

Btw, is it possible `workflowDef` being `undefined`? If so, we need to annotate the function arg 
```diff
- workflowDef: WorkflowDefinition,`
+ workflowDef?: WorkflowDefinition,`
```